igvm/hypervisor_preferences.py

```diff
@@ -101,41 +133,58 @@ def __repr__(self):
 
         return '{}({})'.format(type(self).__name__, args)
 
-    def __call__(self, vm, hv):
+    def get_preference(self, vm, hv) -> Union[float, bool]:
         value = hv.dataset_obj[self.attribute]
 
-        return value is not None, value
+        # if there is no value we assume it's a fresh hv
+        if value is None:
+            return 1.
 
+        # normalize the value. this is only valid for "exhaution values" like
+        # cpu_util_pct and iops_avg
+        return 1 - (value / 100)
```

Comment:
If there is a slight chance that this becomes a integer we should better cast it to float.

Reply:
done